package.json

```diff
@@ -17,7 +17,7 @@
   },
   "dependencies": {
     "@types/assertion-error": "1.0.30",
-    "@types/chai": "3.4.34",
+    "@types/chai": "3.5.2",
```

Comment:
no, but the types haven't changed between this version and 4.0. we could submit a PR for bumping the version